apis/v1alpha3/zz_generated.deepcopy.go

```diff
@@ -1,4 +1,3 @@
-//go:build !ignore_autogenerated
```

Comment:
either restore this line manually, or install go 1.17 on your machine.

Reply:
ok,  I will upgrade to 1.17 and check.

Reply:
Upgrading to Go 1.17 fixed the issue. Thanks!